### PR TITLE
Terraform sample for a quickstart for Azure Virtual Network - Create a vnet with two subnets

### PR DESCRIPTION
**Pull request by asudbring**

### CODE REVIEW

File: quickstart/101-virtual-network-create-two-subnets/variables.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  description = "Location of the resource group."
+}
+
+variable "resource_group_name_prefix" {
+  type        = string
+  default     = "rg"
+  description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+}
+
+variable "vnet_address_space" {
```
Comment (lonegunmanb) [inline quickstart/101-virtual-network-create-two-subnets/variables.tf]: It seems like an unused variable, right?

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Thanks @asudbring for opening this pr! One review comment so far.

File: quickstart/101-virtual-network-create-two-subnets/variables.tf (hunk 1 above)
Comment (asudbring) [reply quickstart/101-virtual-network-create-two-subnets/variables.tf]: Yep, you are right.  I removed and sent up the changes.

**Review by lonegunmanb — APPROVED** [review]
Thanks @asudbring, LGTM!

**Comment by asudbring** [comment]
> Thanks @asudbring, LGTM!

Thank you!!!!!

**Pull request merged**